pointcept/engines/hooks/misc.py

```diff
@@ -16,10 +16,51 @@
 from pointcept.utils.comm import is_main_process, synchronize
 from pointcept.utils.cache import shared_dict
 
+from pointcept.datasets import build_dataset
+from pointcept.datasets.utils import collate_fn
+from pointcept.engines.test import TEST
+
 from .default import HookBase
 from .builder import HOOKS
 
 
+@HOOKS.register_module()
+class AfterTrainTester(HookBase):
+    def __init__(self, test_last=False):
+        self.test_last = test_last
```

Comment:
consider rename the param as: `weight="model_last"`,
consider add a new param: `tester=None` and we will move the tester config into hook;
consider add a new param: `before_train=False` since we can start a test process only by the hook in a future version;